File: Bee-Crowd/Exercises_Python/1036.py
a, b, c = input().split()

a = float(a)
b = float(b)
c = float(c)

delta = (b * b) - (4 * a * c)

if (delta > 0) and (a != 0):
    print('R1 = {:.5f}'.format((- b + (delta ** 0.5)) / (2 * a)))
    print('R2 = {:.5f}'.format((- b - (delta ** 0.5)) / (2 * a)))

else: 
    print('Impossivel calcular')

File: Bee-Crowd/Exercises_C++/1036.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main(){

    double a,b,c;
    cin >> a >> b >> c;

    double delta;
    delta = (b * b) - 4 * a * c;

    if (delta < 0) {
        cout << "Impossivel calcular" << endl;
    }
    else if (a == 0) {
        cout << "Impossivel calcular" << endl;
    }
    else {
            double r1;
            r1 = (-b + sqrt(delta)) / (2.0 * a);

            double r2;
            r2 = (-b - sqrt(delta)) / (2.0 * a);

            cout << fixed << setprecision(5);
            cout << "R1 = " << r1 << endl;
            cout << "R2 = " << r2 << endl;
    }

}